client/app/requests/requests-state.service.js

```diff
@@ -1,8 +1,19 @@
 /** @ngInject */
-export function RequestsStateFactory(ListConfiguration) {
-  var service = {};
+export function RequestsStateFactory(ListConfiguration, RBAC) {
+  var service = {
+    getPermissions: getPermissions,
+  };
 
   ListConfiguration.setupListFunctions(service, { id: 'requested', title: __('Request Date'), sortType: 'numeric' });
 
+  function getPermissions() {
```

Comment:
I looked at it and it looks like those permissions exist for an admin, so I added them in.  It may technically be duplicitous but we can just be explicit in our permissions.